stdlib/experimental/dynamic/dynamic.flux

```diff
@@ -0,0 +1,17 @@
+// Package dynamic provides tools for working with values of unknown types.
+//
+// ## Metadata
+// introduced: NEXT
+//
+package dynamic
+
+
+// dynamic wraps a value so it can be used as a `dynamic` value.
+//
+// ## Parameters
+// - v: the value to wrap
+//
+// ## Examples
+//
+// TODO(onelson): need to do more work before we can really demonstrate this
```

Comment:
You could probably add the `type-conversions` tag to this function:

```
// ## Metadata
// tags: type-conversions
```
```

Reply:
Added :+1: